### ISSUE TITLE
Enhancing Hammer2 to handle collection trees.

### ISSUE THREAD

**Issue by RussellMcOrmond** [crkn-rcdr/Access-Platform#115]
Data in `access` database has been enhanced to support collection trees.  See: https://github.com/crkn-rcdr/CIHM-Meta/issues/33

Hammer2 needs to support these trees in its extracting of old-platform style 'tags'.  This type of functionality is expected to be added to Accessor, and made available to be used by Perl tools such as Hammer2 via #81

In the interim, Hammer2 will be generating incorrect tags, which for any manifests/collections in the tree will mean they won't be accessible on the appropriate demo portal.

**Comment by RussellMcOrmond** [comment]
This can be assigned now if it will be stand-alone (IE: algorithm to get full list of tags is handled within Hammer2), or has a Lapin dependency.  If there is a Lapin dependency, then the feature being added to Lapin should have a ticket and be listed in this ticket.

It may be faster/easier to have this handled internal to Hammer, but will leave that decision to @SaschaAdler 

Algorithm is fairly simple:  Keep looking up any newly added tags for collections they are members of, until nothing new is added.


Example using tree that Sascha created in #33.

* A manifest in the "news" collection would have added "news" as new.
* Looking up "news" would add "serials" as new.
* Looking up "serials" would find "online" as new.
* Looking up "online" would not find anything new, so stop.

**Comment by SaschaAdler** [comment]
Let's start by sorting it out in Hammer, and then re-evaluate later.

**Comment by RussellMcOrmond** [comment]
Seems that Hammer2 hadn't been run on the collections since the collection tree was added to `loadseries`, so none of the documents appeared to be incorrect.

I forced an update on http://iris.tor.c7a.ca:5984/_utils/index.html#database/internalmeta2/oocihm.8_04019 , and it is now missing the collection tags I expected it to.


TODO: Update `loadseries` to adjust the `updateInternalmeta` field.
